test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -197,6 +197,22 @@ def test_sliding_window_cmn(self, center, norm_vars):
             F.sliding_window_cmn, spectrogram, center=center,
             norm_vars=norm_vars)
 
+    @parameterized.expand([("sinc_interpolation"), ("kaiser_window")])
+    def test_resample_waveform(self, resampling_method):
+        num_channels = 3
+        sr = 16000
+        new_sr = sr // 2
+        waveform = common_utils.get_whitenoise(sample_rate=sr, duration=0.5,)
+
+        multi_sound = waveform.repeat(num_channels, 1)  # (num_channels, 8000 smp)
```

Comment:
`get_whitenoise` has `n_channels` option that generates different waveforms for each channel.

https://github.com/pytorch/audio/blob/fad19fabd036879fc3e6a78d544b94c4440c1666/test/torchaudio_unittest/common_utils/data_utils.py#L42

Reply:
Using `n_channels` produces a new input tensor which is failing for linux cpu batch consistency unittests (see [here](https://app.circleci.com/pipelines/github/pytorch/audio/6167/workflows/3de77b09-51b5-477a-b969-a83d05954a63/jobs/224472)). Is it okay to increase atol (from 1e-8 to something like 4e-8)?

Reply:
Yeah, sounds good.

Reply:
actually just noticed the original test used [these values](https://github.com/pytorch/audio/pull/1520/files#diff-a779529b921800f7e0a12fdf7289f2915284fbbaae1ca22149b83dc2089bb78eL251), so going to reuse those

Reply:
sounds good.